_includes/templates/install/ubuntu-queue-kafka.md

```diff
@@ -0,0 +1,98 @@
+#### Kafka Installation
+
+[Apache Kafka](https://kafka.apache.org/) is an open-source stream-processing software platform. 
```

Comment:
Kafka installation guide should be based on docker-compose: https://github.com/wurstmeister/kafka-docker
Just mention the version of Kafka that we recommend and probably the KAFKA_CREATE_TOPICS (or maybe this is redundant for monolithic deployments).